ceci/sites/nersc.py

```diff
@@ -110,6 +110,8 @@ def configure_for_mini(self):
             slurm_cpus_on_node = os.environ.get("SLURM_CPUS_ON_NODE")
             if slurm_cpus_on_node == "64":
                 cpus_per_node = 32
+            elif slurm_cpus_on_node == "256":
+                cpus_per_node = 128
```

Comment:
maybe a #pragma: no cover here?

Reply:
I'll go through at some point and add more tests, so I'll skip the pragma to remind me of that later..